src/inversion_scripts/operators/TROPOMI_operator.py

```diff
@@ -584,8 +586,8 @@ def average_tropomi_observations(TROPOMI, gc_lat_lon, sat_ind):
         corners_lat_index = []
 
         for l in range(4):
-            iGC = nearest_loc(longitude_bounds[l], gc_lons)
-            jGC = nearest_loc(latitude_bounds[l], gc_lats)
+            iGC = nearest_loc(longitude_bounds[l], gc_lons, tolerance=max(dlon,0.5))
+            jGC = nearest_loc(latitude_bounds[l], gc_lats, tolerance=max(dlat,0.5))
```

Comment:
Just checking my understanding. Here we use .5 as default for regular inversion runs, but since we are on a 4x5 grid we need to update this accordingly eg to 4 or 5? Wouldn't we want .25 for .25 inversion runs?

Reply:
This update just allows the TROPOMI operators to be both (1) consistent with previous behavior of the IMI (0.5 degree default) and (2) useful for global simulations like 4 x 5. We wouldn't necessarily want to set this to dlon always. An example is a 1 degree wide TROPOMI pixel (admittedly rare). If we search 0.3125 degrees from its corners for GC grid cell centers, we would miss the one at 0.47.
<img width="913" alt="image" src="https://user-images.githubusercontent.com/91984611/234955632-b56fea34-09db-4616-bf71-82533e0e765c.png">

This isn't a perfect fix ([this ](https://github.com/geoschem/integrated_methane_inversion/issues/110) should address it).